docs/navigation.md

```diff
@@ -3,72 +3,26 @@ id: navigation
 title: Navigating Between Screens
 ---
 
-Mobile apps are rarely made up of a single screen. Managing the presentation of, and transition between, multiple screens is typically handled by what is known as a navigator.
+By default, React Native renders your components onto a single screen. To create multiple screens and navigate between them you need a navigation library. The following table helps you choose the one that's right for you. Each column identifies a different feature and each row represents a different navigation library (React Router Native isn't included because it doesn't have stack navigation). These features aren't the be-all and end-all of what makes a good navigation library but they are important. Keep reading to find out why.
 
-This guide covers the various navigation components available in React Native. If you are just getting started with navigation, you will probably want to use [React Navigation](navigation.md#react-navigation). React Navigation provides an easy to use navigation solution, with the ability to present common stack navigation and tabbed navigation patterns on both iOS and Android.
+|                                                                                                           | Uses native API | Has one React root | Runs in Expo | Runs on the Web |
+| --------------------------------------------------------------------------------------------------------- | --------------- | ------------------ | ------------ | --------------- |
+| [React Navigation](https://reactnavigation.org/docs/en/getting-started.html)                              | No              | Yes                | Yes          | No              |
+| [React Native Navigation](https://wix.github.io/react-native-navigation/)                                 | Yes             | No                 | No           | No              |
+| [The Navigation router](https://grahammendick.github.io/navigation/documentation/native/hello-world.html) | Yes             | Yes                | No           | Yes             |
```

Comment:
I'm not sure how to say this without it coming off as unkind, so I am sorry @grahammendick, but this library should not be listed here. As far as I know nobody uses it in production. It may be reasonable to include it at some point in time but it needs to prove itself through usage.

Reply:
I agree. Can you remove it? I have never heard of this library before.

Reply:
Thanks @brentvatne, no offence taken. I hope you won’t take any from what I have to say.

It’s interesting to see React gatekeeping out in the open. It’s obvious it goes on but it’s usually more subtle than that.

If you’re concerned about libraries proving themselves first then why does [React Navigation promote React Native Router](https://reactnavigation.org/docs/en/alternatives.html)? React Router Native doesn’t even have stack navigation because it doesn’t hold visited screens in memory. This is the lowest bar for any native navigation library.

Why didn’t Airbnb’s Native Navigation library have to prove itself? [It was promoted on this page](https://github.com/facebook/react-native-website/pull/532) for ages despite it not being production-ready and having threadbare documentation. The project’s been disbanded.

It’s obvious that React gatekeeping isn’t working. It lets through projects that should be turned away and closes the door to projects that should be welcomed. I wanted to rewrite this page as a corrective to the corrosive nature of React gatekeeping. Is that going to be possible?

Reply:
The current guidance for the React Native website is to list repositories that are either part of the RN community org ( https://github.com/react-native-community ) or have significant adoption. If any library part of the React Native ecosystem could be listed on the website it will confuse people with too many packages and increase maintenance burden for us to check which packages work with the latest RN releases.

Reply:
@cpojer Can you explain what checks you run? Do you plan to update this page to say that React Native Navigation doesn't work with RN 0.60?

Reply:
As you pointed out yourself, even with a small number of libraries mentioned on the website it is still hard to ensure they are all working with the latest versions of React Native. Imagine increasing the amount of libraries mentioned significantly and how that would make people lose trust in the RN documentation.

React Native Navigation is currently being made ready for 0.60: https://github.com/wix/react-native-navigation/issues/5228

Reply:
@grahammendick - there is no gatekeeping happening here, we are just trying to provide a good initial experience for new users and I don't think that guiding them towards an unproven library is a responsible thing to do. I'd appreciate a bit more empathy for our situation rather than assuming that folks are acting in bad faith to keep certain libraries in favor and limit the popularity of others. I'll repeat to you again - as a maintainer of React Navigation, I would love for another library to come along and be better in every way and replace it, then the React Native community would be better and I would have less work to do. I encourage you to keep working on your library, kick the tires in some production apps and get some early adopters.

Reply:
> If you’re concerned about libraries proving themselves first then why does React Navigation promote React Router Native

IMO we should remove it from there

Reply:
> Why didn’t Airbnb’s Native Navigation library have to prove itself? It was promoted on this page for ages despite it not being production-ready and having threadbare documentation. The project’s been disbanded.

It was backed by a big company which already used it in production. So it was fair to assume that they'll keep maintaining it. We can't predict future and we make mistakes in our assumptions.

But I feel it doesn't matter now since we've already removed it when we got to know that it's not maintained anymore.

Reply:
These mistakes only ever happen in one direction. They always promote the powerful. Companies and people whose voices are already amplified in the React community. This is what I mean by gatekeeping @brentvatne . I don’t think you’re acting in bad faith, I think you’re suffering from confirmation bias.

Thanks @satya164  for responding to my specific points. I’ll be very happy if you remove React Router Native because It will help prevent users taking a wrong turn.

Reply:
https://github.com/react-navigation/react-navigation.github.io/commit/921b263d37649f05beb13ca6e2988d457071ae9b

Reply:
> They always promote the powerful. 

I think you are mixing correlation and causation. We try to promote production battle-tested libraries used by many engineers. There is a correlation with those libraries and powerful organizations because powerful organizations usually have lots of engineers and users doing that battle testing. But it's entirely possible for a lone developer to come up with an awesome library, and grow enough early adopters to prove it out, but they need to seed that growth themselves.